docs/spec/spec.md

```diff
@@ -74,11 +74,23 @@ byte[] serializedData = check schema.toAvro("avro-data");
 
 The `data` parameter accepts the following Ballerina data types that is needed to be serialized into `byte` array.
 
-###### 3.1.1.1.1 Accepted Ballerina types
-
-* `json`
-* `enum`
-* `record`
+###### 3.1.1.1.1 Map Avro types to Ballerina Types
+
+The following table summarizes how Avro types are mapped to corresponding Ballerina types. These rules are applicable when serializing/deserializing Ballerina data according to an Avro schema.
+
+| Avro Type    | Ballerina Type | Example                                       |
+|--------------|----------------|-----------------------------------------------|
+| null         | nil            | ()                                            |
+| boolean      | boolean        | true                                          |
+| int,long     | int            | 1                                             |
+| float,double | float          | 1.1                                           |

```

Comment:
It is better to explain why we use `int` and `float` in Ballerina for `long` and `double` types. (Maybe we can point the ballerina spec or just describe the ballerina types here.